reference/cli/admin_utilities/hawqinit.html.md.erb

```diff
@@ -4,7 +4,7 @@ title: hawq init
 
 The `hawq init cluster` command initializes a HAWQ system and starts it.
 
-The master or segment nodes can be individually initialized by using `hawq init master` and `hawq init segment` commands, respectively. The `hawq init standby` command initializes a standby master host for a HAWQ system.
+The master or segment nodes can be individually initialized by using `hawq init master` and `hawq init segment` commands, respectively.  Format options can also be specified at this time. The `hawq init standby` command initializes a standby master host for a HAWQ system.
```

Comment:
Small edit: Change "initialized by using" to "initialized by using the"

Also edit for passive voice here.